DRAFT_RELEASE_NOTES.md

```diff
@@ -75,3 +75,7 @@ To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.9.0
 1. Repeat the previous step until all manual actions are resolved
 
 # What's Changed
+
+- MLFlow Version Upgrade: We have updated the MLFlow chart version to 1.4.22 in this release, bringing enhancements and stability improvements to the platform. 
```

Comment:
Q: Do we need to upgrade any libraries for the new version of MLFlow?

Reply:
No, everything was handled through the Docker image.